tests/scancode/test_cli.py

```diff
@@ -134,6 +134,15 @@ def test_license_option_detects_licenses():
     assert len(open(result_file).read()) > 10
 
 
+def test_can_call_run_scan_as_a_function():
+    from scancode.cli import run_scan
+    test_dir = test_env.get_test_loc('license', copy=True)
+    rc, results = run_scan(test_dir, license=True, copyright=True, return_results=True)
+    assert rc
+    assert len(results['files']) == 2
+    assert not results['headers'][0]['errors']
+
```

Comment:
@pombredanne I think this test is showing wrong results.

Reply:
How so? this is to ensure that a scancode can be called as a function and that we get no errors there. This was added for #1780 with https://github.com/nexB/scancode-toolkit/commit/51596ba19c35ab69440edde7da02ef89c6eb99c4#diff-bcef54cd07fd9a0279da1c83afd9a0b4R129 and removing it will likely be a source of regression. If you have something against it be provide a detailed reason that you may have: I want to make that I am not discarding your concern if this is properly substantiated.